### PR TITLE
Update the gcs chunk_size documentation.

### PR DESCRIPTION
**Pull request by cafuego**
Remove `1g` from the examples, as the GCS repository chunk_size can be at most 100m. As illustrated by the error message when trying to set it to `1g`:

```
$ curl -H 'Content-type: application/json' -u user:pass -XPUT http://127.0.0.1:9200/_snapshot/backup -d '{
  "type": "gcs",
  "settings": {
    "bucket": "elk-backup",
    "client": "backups",
    "chunk_size": "1g"
  }
}'
{"error":{"root_cause":[{"type":"repository_exception","reason":"[backup] failed to create repository"}],"type":"repository_exception","reason":"[backup] failed to create repository","caused_by":{"type":"illegal_argument_exception","reason":"failed to parse value [1g] for setting [chunk_size], must be <= [100mb]"}},"status":500}
```

### CODE REVIEW

File: docs/plugins/repository-gcs.asciidoc [hunk 1]
```diff
@@ -232,7 +232,7 @@ The following settings are supported:
 
     Big files can be broken down into chunks during snapshotting if needed.
     The chunk size can be specified in bytes or by using size value notation,
-    i.e. `1g`, `10m`, `5k`. Defaults to `100m`.
+    i.e. `10m` or `5k`. Defaults to `100m`, which is the maximum permitted.
```
Comment (original-brownbear) [inline docs/plugins/repository-gcs.asciidoc]: I know it was like that before, but it's just wrong. Could you replace `i.e.` by `e.g.` here, please?
Other than that, this looks fine to me.  I added a todo for myself though to look into this limitation (it's coming from our code, not from GCS itself as far as I can tell), it seems unnecessary potentially.

**Review by original-brownbear — CHANGES_REQUESTED** [review]
One small change please :) Thanks for spotting this!

**Comment by original-brownbear** [comment]
@cafuego can you sign the CLA please as well? Thanks!

**Comment by cafuego** [comment]
@original-brownbear I have already signed the CLA, I have a PDF to prove it :-)  The github username on the form looks ok to me, so not sure what went wrong there.

**Comment by original-brownbear** [comment]
@cafuego 

> I have already signed the CLA, I have a PDF to prove it :-)

Did you use the same email address for signing the CLA that is used by your git commits here? Also, is that email address connected to your Github account. It's probably one of these two issues.

**Comment by cafuego** [comment]
I checked, yes the email address I used fo the CLA is the same as the one. However, I did a quick edit via the GitHub UI and the committer is noreply@github.com, which is probably what's wrong. I did a history rewrite on my commits and force pushed from local, but that has made GitHub close the PR. I'll see if I can re-open it (probably as a new PR, sorry)

**Comment by original-brownbear** [comment]
@cafuego no worries, ping me in the new PR and I'll take a look in a few hours in my morning :) Thanks!

**Pull request closed without merging**